### ISSUE TITLE
MODIFIER MONTANT FACTURE DANS LETTRAGE

### ISSUE THREAD

**Issue by josneaulgi** [lgi-wines/Extranet-LGI#18]
Si on modifie le montant d'une facture, il faudrait que ce montant se mette à jour automatiquement dans le lettrage.
http://extranet.lgi-wines.com/Client/Letter.aspx?ClientID=127

**Comment by josneaulgi** [comment]
RELANCE

**Comment by josneaulgi** [comment]
RELANCE

**Comment by MamboJoel** [comment]
Il s'agit donc de recalculer l'échéancier. Celui-ci peut comporter plusieurs échéances. Et techniquement, rien n'empêche qu'une échéance ait été recouvrée déjà. Que se passe-t-il lorsque, dans ce cas, l'on change le montant de la facture ?

**Comment by josneaulgi** [comment]
non pas du tout. Il ne s'agit pas de recalculer l'échéancier. C'est si on modifie le montant de la facture, il faudrait que ce montant soit modifié automatiquement dans le lettrage.
http://extranet.lgi-wines.com/Client/Letter.aspx?ClientID=127

**Comment by MamboJoel** [comment]
Pourtant le lettrage consiste à affecter des virements (débit) à des échéances (crédit) issues de l'échéancier :

![image](https://cloud.githubusercontent.com/assets/3832009/23657229/5b9912e8-033d-11e7-9dc4-3d6562184638.png)

**Comment by josneaulgi** [comment]
tout à fait. par "échéancier", je pensais à date d'échéance et pas le montant de l'échéancier. Maintenant que j'ai compris ton raisonnement, c'est ça.

**Comment by MamboJoel** [comment]
Ok, alors ma question quant à l'échéancier :

> Celui-ci peut comporter plusieurs échéances. Et techniquement, rien n'empêche qu'une échéance ait été recouvrée déjà. Que se passe-t-il lorsque, dans ce cas, l'on change le montant de la facture ?

**Comment by josneaulgi** [comment]
techniquement, non car chez LGI pas de départ, pas de facture donc pas d'échéance.  Et quand bien même, une partie de la commande serait déjà recouvrée, on délettrerait.
